paths/git-trouble/fc-scenarios.md

```diff
@@ -0,0 +1,34 @@
+---
+layout: simple-class
+header:
+  overlay_image: cover.jpeg
+  overlay_filter: rgba(46, 129, 200, 0.6)
+title: Scenarios
+permalink: /git-trouble/02
+facilitator: false
+sidebar:
+  nav: "advanced"
+main-content: |  
+  ### [Too Many (small) Commits!](/on-demand/git-trouble/03)
```

Comment:
@beardofedu as I first looked at this page, it didn't immediately jump out to me that these were different scenarios I should choose from. Do you think we can do something at the top of the page to make that more clear? Admittedly, the "Scenarios" in the header is completely invisible to me once I start the course.